.travis.yml

```diff
@@ -32,6 +32,7 @@ script:
     - python -m unittest discover -s jenkins/test-history -p "*_test.py"
     - ./jenkins/bootstrap_test.py
     - ./jenkins/diff-job-config-patch.sh
+    - ./jenkins/diff-e2e-runner.sh
```

Comment:
Not too worried, diff-e2e-runner.sh just figures out which images it needs to compare and then invokes Python for the complex stuff.